tests/tests_integration/test_api/test_postgres_gateway/test_tables.py

```diff
@@ -0,0 +1,128 @@
+import pytest
+from jaraco.context import suppress
+
+from cognite.client import CogniteClient
+from cognite.client.data_classes._base import UnknownCogniteObject
+from cognite.client.data_classes.data_modeling import (
+    Container,
+    ContainerApply,
+    ContainerProperty,
+    MappedPropertyApply,
+    Space,
+    SpaceApply,
+    Text,
+    View,
+    ViewApply,
+)
+from cognite.client.data_classes.postgres_gateway import (
+    Column,
+    RawTableOptions,
+    RawTableWrite,
+    Table,
+    TableList,
+    User,
+    ViewTableWrite,
+)
+from cognite.client.exceptions import CogniteAPIError
+from cognite.client.utils._text import random_string
+
+
+@pytest.fixture
+def one_space(cognite_client: CogniteClient) -> Space:
+    my_space = SpaceApply(
+        space=f"my_space-{random_string(10)}",
+    )
+    created = cognite_client.data_modeling.spaces.apply(my_space)
+    yield created
+    cognite_client.data_modeling.spaces.delete(created.space)
+
+
+@pytest.fixture
+def one_container(cognite_client: CogniteClient, one_space: Space) -> Container:
+    my_container = ContainerApply(
+        space=one_space.space, external_id="my_container", properties={"name": ContainerProperty(type=Text())}
+    )
+    created = cognite_client.data_modeling.containers.apply(my_container)
+    yield created
+    cognite_client.data_modeling.containers.delete(created.as_id())
+
+
+@pytest.fixture
+def one_view(cognite_client: CogniteClient, one_space: Space, one_container: Container) -> View:
+    my_view = ViewApply(
+        space=one_space.space,
+        external_id="my_view",
+        version="v1",
+        properties={"name": MappedPropertyApply(one_container.as_id(), "name")},
+    )
+    created = cognite_client.data_modeling.views.apply(my_view)
+    yield created
+    cognite_client.data_modeling.views.delete(created.as_id())
+
+
+@pytest.fixture
+def one_raw_table(cognite_client: CogniteClient) -> tuple[str, str]:
+    db_table_pair = f"my_database-{random_string(10)}", "my_table_postgres_gateway"
+
+    cognite_client.raw.databases.create(db_table_pair[0])
+    cognite_client.raw.tables.create(*db_table_pair)
+
+    yield db_table_pair
+    with suppress(CogniteAPIError):
+        cognite_client.raw.tables.delete(*db_table_pair)
+        cognite_client.raw.databases.delete(db_table_pair[0])
```

Comment:
If the first line raises `CogniteAPIError`, the second line will never be run. Is that intended?